Godeps/_workspace/src/github.com/GoogleCloudPlatform/kubernetes/pkg/client/helper.go

```diff
@@ -261,6 +263,20 @@ func TransportFor(config *Config) (http.RoundTripper, error) {
 			transport = http.DefaultTransport
 		}
 	}
+
+	if glog.V(6) {
+		transport = NewDebuggingRoundTripper(transport, URLTiming)
+	}
+	if glog.V(7) {
+		transport = NewDebuggingRoundTripper(transport, JustURL, RequestHeaders, ResponseStatus)
+	}
+	if glog.V(8) {
+		transport = NewDebuggingRoundTripper(transport, JustURL, RequestHeaders, RequestBody, ResponseStatus, ResponseHeaders, ResponseBody)
+	}
+	if glog.V(9) {
```

Comment:
> what are two more things we could show? can it go to 11?

In fairness, I'd really like to determine what to add based on flags, but that takes more plumbing than I think we want to do.